tests/test_strategy_cpu.py

```diff
@@ -79,6 +87,9 @@ def test_implicit(tmpdir):
     _run_horovod(trainer_options)
 
 
+@pytest.mark.xfail(
+    raises=RuntimeError, reason="Training with multiple optimizers is only supported with manual optimization"
+)
```

Comment:
The BasicGAN needs to be rewritten with manual optimization. You can copy paste BasicGAN from lightning under tests_pytorch/helpers/advanced_models.py

Reply:
yes, that is why I will make it as xfail not just silent drop :)

Reply:
I wouldn't make it as xfail. The simplest is to copy-past the BasicGAN over to tests/helpers.

Reply:
thete is already one `xfail` but pytest does not allow mark one tests with two so far I know...

Reply:
found it... https://stackoverflow.com/a/38403866/4521646
but still we do not expect user using this with aligning below 2.0, right?